src/shared/components/VoteButton.tsx

```diff
@@ -1,12 +1,47 @@
 import React, { PureComponent } from "react";
 
 interface Props {
-  active: Boolean;
+  active: boolean;
+  clickHandler: () => void;
 }
 
-export default ({ active }: Props) => (
-  <svg width="24" height="24" viewBox="0 0 24 24" className={"voteButton"}>
-    <path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" />
-    <path d="M0 0h24v24H0z" fill="none" />
-  </svg>
-);
+interface State {
+  currentAnimation: string;
+}
+
+export default class extends PureComponent<Props, State> {
+  state = {
+    currentAnimation: ""
+  };
+
+  onClick = () => {
+    this.setState({ currentAnimation: "" });
```

Comment:
Do you need to set state here?

Reply:
Yeahhh, cause we're using css class animations, need to remove the class and re-add it to perform the animation